### ISSUE TITLE
Multiple Databases?

### ISSUE THREAD

**Issue by andrewteg** [CrunchyData/pg_tileserv#97]
We have multiple databases on a shared server that we'd like to setup with pg_tileserv.  I can do with multiple instances by changing the port on config files right now, but I'm wondering if there is any way to use the same port and change the BasePath or have some more uniform solution.  Right now I'm just starting two instances for testing like:
```
pg_tileserv.exe -c config/db1.toml
pg_tileserv.exe -c config/db2.toml
```
Is this the best way or is there a better way?

If it matters, it's a Windows server for now (Linux eventually) and my hope is to make whatever final solution a service or some other always-on option.

Thanks!

**Comment by HardRock4Life** [comment]
@andrewteg What do you mean on the BasePath? Is it like changing an ip address?

**Comment by yoad-ashuri** [comment]
hi :)
are you planning to do this great idea of @andrewteg with the Baseurl ( http://localhost:7800/db1/ and http://localhost:7800/db2/)?

I think this is good practice don’t you think?

thanks in advance,
love your product!

**Comment by pramsey** [comment]
No, not planning on it. Can't really use the "same connection on multiple databases" as that implies "same user on multiple databases" which isn't a great pattern necessarily. The idea is to keep the deployment very simple, and "one connection string per instance" certainly fits that.

**Comment by FaFre** [comment]
I'm at a similar point now, I have three different layers on three different database servers, and would like to query them in on go, like it is possible using the comma delimiter. I know there are some possible workarounds for PostgreSQL like `dblink` or foreign wrapper, but I don't think those are ideal. 

Do you guys have an idea on how to handle this?
